app/services/user.ts

```diff
@@ -86,7 +84,8 @@ export class UserService extends PersistentStatefulService<IUserServiceState> {
     // actually log in from integration tests.
     electron.ipcRenderer.on('testing-fakeAuth', async (e: Electron.Event, auth: IPlatformAuth) => {
       const service = getPlatformService(auth.platform.type);
-      this.login(service, auth);
+      await this.login(service, auth);
+      this.onboardingService.next();
```

Comment:
what we skip here?

Reply:
We actually don't skip anything, we just need to continue from the login step, since simply setting the login state isn't enough.